ultimate_hosts_blacklist/whitelist/__init__.py

```diff
@@ -277,14 +278,16 @@ def _command_line():
         "--whitelist",
         type=argparse.FileType("r"),
         nargs="+",
-        help="Read the given file and append its data to the our whitelist list.",
+        help="Read the given file and append its data to the UHBW's\n"
+        "hosted whitelist list.",
     )
 
     parser.add_argument(
-        "-wc",
+        "-wc\n",
```

Comment:
@spirillen, linebreak (`\n`) is not authorized with `-wc`. By doing so, you are killing the whole switch!

Reply:
f....

Reply:
Believes I've cleaned up those I had made. See also keybase